packages/components/ContextualMenu/src/ContextualMenu.tsx

```diff
@@ -43,9 +43,11 @@ export const ContextualMenu = compose<ContextualMenuType>({
 
     React.useLayoutEffect(() => {
       if (Platform.OS === 'macos') {
```

Comment:
```=== 'macos' && shouldFocusOnMount```

Reply:
Separate file retro-review + unsure -- is a Slot actually necessary for the FocusZone, versus making it part of the ```Container``` Slot render? Pattern is repeated in SubMenu as well.

Also FocusZone related clarification -- does the parent FocusZone capture key events in the case of wanting to have horizontal focus internally e.g. if a menu item was desired w/ horizontal navigation contents in an otherwise normal context menu?

Reply:
Container is just a View, I don't really have a way to make it a View on win32 and a FocusZone on macOS. I don't really have access to the Container slot's render either, since it's just a View. 
Nit: In every other control, the Root Slot is a View. In ContextualMenu, _Container_ is a View, while _Root_ is a Callout. That seems wrong to me, and I'd like to swap that naming for V1. I haven't yet because that could be a breaking change.

FocusZone on macOS captures key events, and lets them pass through if the key event isn't one it handles. In your case, I'm imagining a menu item with a row of focusable views. I think by defualt, `Left/Right Arrow` will move focus up and down. That behavior might be disable-able with a prop, which is what we would have to do to support that scenario.